src/pages/Task/TaskRunTable-Tile.vue

```diff
@@ -71,7 +72,11 @@ export default {
       return `%${this.searchTerm}%`
     }
   },
-  methods: {},
+  methods: {
+    durationCalc(startTime, endTime, state) {
```

Comment:
nit: I would import the method with this full name `calculateDuration`, and rather than using another name here but calling the imported method I would just make it available to the template with

```
methods: {
    calculateDuration
}
```

which should work fine since the arguments do not change